### DISCUSSION TITLE
Unable to Print Receipt After Executing "Declare Start Amount" Operation Using POS API

### DISCUSSION THREAD

**Discussion by tatjotta** [microsoft/Dynamics365Commerce.Solutions#278] [Q&A]
I am developing a new interface with functionality equivalent to the CashManagementView. I have successfully used CreateStartingAmountTransactionClientRequest to update the Amount, but when I attempt to print the receipt afterward, I encounter a server error. The error message states: "The request cannot be processed at the moment. Please try again later. Status code 500."

I am confident that the ReceiptType of 'StartingAmount' is correct. The properties IsCopy, IsRemoteTransaction, IsPreview, and QueryBySalesId are all set to false.

Does anyone know how I might resolve this issue to successfully execute the receipt printing process? Any insights would be greatly appreciated.

**Comment by zhangguanghuib** [comment]
Can you try this code, it worked for anther customer? 

https://github.com/zhangguanghuib/NewCommerceSDK/blob/main/KnowledgeArticles-Development/4%20How%20to%20print%20Custom%20Type%20Receipt.md

**Reply by tatjotta** [reply]
Thank you. I was able to resolve the issue. By correctly defining the parameters during the execution of CreateStartingAmountTransactionClientRequest, I was able to print the desired Receipt.
